### PR TITLE
Updates for iBVP Dataset and iBVPNet Model

### PR DESCRIPTION
**Pull request by jnj256**
**iBVP Dataset:**
- Updated iBVP dataloader to support thermal video frames.
- Updated support for additional signal quality labels from second annotator (recommend to re-download the dataset for this change, if dataset is downloaded before 14-June-2024).

**iBVPNet Model:**
- Updated iBVPNet to take input as raw video frames, with Diff implemented within the network and correspondingly updated its trainer. 
- Releasing updated pretrained model for iBVPNet trained on PURE dataset, and EfficientPhys trained on iBVP dataset

**Other minor changes:**
- Removed "from scipy.__config__ import get_info" from MMPDLoader, as newer Scipy versions throw error here.
- In "CHROME_DEHAAN.py" changed 'signal.hanning' to more recently supported 'signal.windows.hann'.

### CODE REVIEW

File: unsupervised_methods/unsupervised_predictor.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ def unsupervised_predict(config, data_loader, method_name):
         batch_size = test_batch[0].shape[0]
         for idx in range(batch_size):
             data_input, labels_input = test_batch[0][idx].cpu().numpy(), test_batch[1][idx].cpu().numpy()
+            data_input = data_input[..., :3]
```
Comment (yahskapar) [inline unsupervised_methods/unsupervised_predictor.py:30]: Before merging, can you double-check to make sure this change doesn't affect other datasets (e.g., UBFC-rPPG, PURE) and how the raw input might be formatted? I wouldn't think there's any difference, but just in case I'd be sure to double-check.
